src/components/TimelineMessage.tsx

```diff
@@ -385,55 +401,93 @@ export function TimelineMessage(props: TimelineMessageProps): JSX.Element {
-                                        color: theme.palette.text.secondary
-                                    }}
-                                    color="primary"
-                                    onClick={() => {
-                                        unfavorite(
-                                            message?.id,
-                                            message?.associations_data.find(
-                                                (e) =>
-                                                    e.author ===
-                                                    appData.userAddress
-                                            )?.id
-                                        )
-                                    }}
-                                >
-                                    <StarIcon />{' '}
-                                    <Typography sx={{ size: '16px' }}>
-                                        {
-                                            message.associations_data.filter(
-                                                (e) => e.schema === Schemas.like
-                                            ).length
-                                        }
-                                    </Typography>
-                                </IconButton>
-                            ) : (
-                                <IconButton
-                                    sx={{
-                                        p: '0',
-                                        color: theme.palette.text.secondary
-                                    }}
-                                    onClick={() => {
-                                        favorite(message?.id)
-                                    }}
-                                >
-                                    <StarOutlineIcon />{' '}
-                                    <Typography sx={{ size: '16px' }}>
-                                        {
-                                            message.associations_data.filter(
-                                                (e) => e.schema === Schemas.like
-                                            ).length
-                                        }
-                                    </Typography>
-                                </IconButton>
-                            )}
+                            <Tooltip
+                                title={
+                                    <Box
+                                        sx={{
+                                            display: 'flex',
+                                            flexDirection: 'column',
+                                            gap: 1
+                                        }}
+                                    >
+                                        {reactUsers.map((user) => (
+                                            <Box
+                                                key={user.username}
```

Comment:
```suggestion
                                                key={user.ccaddress}
```
usernameはかぶる可能性があるのでkeyとしてはccaddressがいいカモ